app/services/riiif/region/percentage.rb

```diff
@@ -19,49 +26,49 @@ def initialize(image_info, x, y, width, height)
       # @return [String] a region for imagemagick to decode
       #                  (appropriate for passing to the -crop parameter)
       def to_imagemagick
-        "#{@width_pct}%x#{@height_pct}+#{offset_x}+#{offset_y}"
+        "#{width_pct}%x#{height_pct}+#{offset_x}+#{offset_y}"
       end
 
       def maintain_aspect_ratio?
-        @width_pct == @height_pct
+        width_pct == height_pct
       end
 
       private
 
-        # @param [String] n a percentage to convert
+        # @param [Float] n a percentage to convert
         # @return [Float]
         def percentage_to_fraction(n)
-          Integer(n).to_f / 100
+          n / 100
```

Comment:
Out of an abundance of caution, maybe this should be `100.0` to avoid any unexpected behavior if an integer is passed in instead of a float?